azure-pipelines.yml

```
@@ -32,7 +32,18 @@ jobs:
     - script: sudo apt-get install -qy --force-yes graphviz
       displayName: 'Install GraphViz'
 
-    - script: make html
+    - script: |
+        wget -O drawio.deb https://github.com/jgraph/drawio-desktop/releases/download/v12.6.5/draw.io-amd64-12.6.5.deb
+        sudo apt-get install libasound2 ./drawio.deb
+      displayName: 'Install draw.io'
+
+    - script: sudo apt-get install xvfb
```

```suggestion
        sudo apt-get install -qy --force-yes xvfb
```
this is what the other commands do

---

same as other one